### PR TITLE
[QRF-350] Fix typo in Periodicity.BI_ANUALLY enum value

### PR DESCRIPTION
**Pull request by oleg-semyonov**

### CODE REVIEW

File: src/types/Contact.ts [hunk 1]
```diff
@@ -24,7 +24,7 @@ export enum Periodicity {
     BI_MONTHLY = 'bi-monthly',
     QUARTERLY = 'quarterly',
     ANNUALLY = 'annually',
-    BI_ANUALLY = 'bi-anually',
+    BI_ANNUALLY = 'bi-annually',
```
Comment (e1himself) [inline src/types/Contact.ts:27]: > `BI_ANNUALLY`

The constant name cannot be renamed under 7.x. As it's a BC break. Let's only change the value here.

Comment (oleg-semyonov) [reply src/types/Contact.ts:27]: Oh, right, my bad

Comment (oleg-semyonov) [reply src/types/Contact.ts]: https://github.com/prezly/javascript-sdk/pull/160/commits/039ce32f80a029ffd39b1dec0ec43eca57341c31

**Review by e1himself — APPROVED** [review]

**Comment by e1himself** [comment]
This patch has been propagated to 8.x and 9.x as: v8.0.1 and v9.0.2

**Pull request merged**